### ISSUE TITLE
[Bug Report][2.6.5] NVDA is not announcing user selection when users made a selection from v-select dropdown

### ISSUE THREAD

**Issue by JackLeiLing** [vuetifyjs/vuetify#15033]
### Environment
**Vuetify Version:** 2.6.5
**Vue Version:** 2.6.1
**Browsers:** Chrome 100.0.4896.127
**OS:** Mac OS 10.15.7

### Steps to reproduce
When a screen reader (NVDA) user has made a selection from the ‘Enrolling for’ and 'School year to be enrolled' in dropdowns, the item they have selected is not announced, and the incorrect element role is announced. Example: “Enrolling for, edit select. Blank.” Or, when a screen reader user uses arrow keys to navigate through the menu items without expanding the list items, nothing is announced.

### Expected Behavior
expected the NVDA to announce the user's selection; options are announced when users are navigating through the options

### Actual Behavior
the item they have selected is not announced, and the incorrect element role is announced. Example: “Enrolling for, edit select. Blank.” 

### Reproduction Link
<a href="https://codepen.io/jackleiling/pen/dydoyoO" target="_blank">https://codepen.io/jackleiling/pen/dydoyoO</a>

### Other comments
it could be caused by the hidden input without value after the selection text
<div class="v-input__control">
  <div role="button" aria-haspopup="listbox" aria-expanded="false" aria-owns="list-47" class="v-input__slot">
    <fieldset aria-hidden="true">
      <legend style="width: 105.75px;"><span>​</span></legend>
    </fieldset>
    <div class="v-select__slot">
      <label for="input-47" class="v-label v-label-active themelight primary-text" style="left: 0px; right: auto; position: absolute;">Enrolling for</label>
      <div class="v-select__selections">
        <div class="v-select_selection v-select_selection--comma">2022 calendar year</div>
        <input data-testid="calendar-year" id="input-47" placeholder="Select..." readonly="readonly" type="text" aria-readonly="false" autocomplete="off" aria-activedescendant="list-item-66-0">
    </div>
  <div class="v-input__append-inner">
    <div class="v-input_icon v-input_icon--append">
      <i aria-hidden="true" class="v-icon notranslate mdi mdi-menu-down theme-light primary-text"></i>
    </div>
  </div>
    <input type="hidden" value="2022">
  </div>
  <div class="v-menu"></div>
</div>

**Comment by KaelWD** [comment]
Possibly related to #12454

**Comment by RyanSMS** [comment]
I think this related closely with https://github.com/vuetifyjs/vuetify/issues/15423
The reason that NVDA read blank even in the case there is selected value from the control is at the structure design of v-select control - when focus, the empty textbox gets focused, so it reads as "edit, blank".
